client/components/LearnerViewDirective.js

```diff
@@ -192,26 +205,52 @@ tie.directive('learnerView', [function() {
       </style>
     `,
     controller: [
-      '$scope', 'SolutionHandlerService', 'QuestionDataService',
+      '$scope', '$timeout', 'SolutionHandlerService', 'QuestionDataService',
       'LANGUAGE_PYTHON',
       function(
-          $scope, SolutionHandlerService, QuestionDataService,
+          $scope, $timeout, SolutionHandlerService, QuestionDataService,
           LANGUAGE_PYTHON) {
         var language = LANGUAGE_PYTHON;
-        var question = QuestionDataService.getData();
-        var prompts = question.getPrompts();
-        var currentPromptIndex = 0;
+        var questionSetId = 'strings';
```

Comment:
I think we don't need a TODO for something like this -- in my experience, TODOs tend to rot (since there's no good system for keeping track of them well) and I would lean towards not having them where they aren't necessary. In the future, when we expand the requirements of the app to "handle multiple question sets", this will definitely need to be changed anyway -- it isn't a subtle thing that would otherwise go ignored.